pkg/util/iptables/iptables.go

```diff
@@ -581,6 +581,8 @@ func getIPTablesWaitFlag(version *utilversion.Version) []string {
 func getIPTablesRestoreWaitFlag(version *utilversion.Version) []string {
 	if version.AtLeast(WaitRestoreMinVersion) {
 		return []string{WaitString, WaitSecondsValue}
+	} else if version.String() == "1.4.21" { // HACK for 1.16.0; see https://github.com/kubernetes/kubernetes/issues/82587
```

Comment:
Is there ANY other way to cross check this against something else to filter out false-positives?

Reply:
I was trying to minimize the amount of new code. The right way to test would be to bring back the original code that ran `iptables-restore --version` to see how it behaves. That would look like https://github.com/danwinship/kubernetes/commit/75888077, which I've confirmed passes unit tests but haven't tested beyond that. Note that all of the "new" code there is just the old code, although this isn't exactly like it was before #80368; it still fixes the original problem from that PR, that on RHEL 7 we were telling `iptables` to wait forever but `iptables-restore` to only wait 5 seconds. So assuming that commit is correct, that's what I'd want to merge post-1.16.0.

If you wanted something with fewer false positives but still very small, maybe we could test "if version is 1.4.21 and /etc/redhat-release exists"?